packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -3645,7 +3775,10 @@ class ErrorWidget extends LeafRenderObjectWidget {
   @override
   void debugFillProperties(DiagnosticPropertiesBuilder properties) {
     super.debugFillProperties(properties);
-    properties.add(StringProperty('message', message, quoted: false));
+    if (_flutterError == null)
+      properties.add(StringProperty('message', message, quoted: false));
+    else
+      _flutterError.debugFillProperties(properties);
```

Comment:
would this look better if it was added as a child property?

Reply:
(i'm curious to see how an ErrorWidget widget renders in the dumped tree)

Reply:
Good point! That looks better. I am using the whitespace style now for the FlutterError now but for fun we could use the Error style to really highlight the error message in the tree however that would require tweaking some rendering in a follow up CL as I support error style children well but not error style properties (it works but isn't up to the polish of the other tree rendering).

The extra indent adds a bit of clarity and that lets us get out of using the tree style used for widgets which isn't great for multiline properties.
 Here is the after output for an error:
```
flutter:                               │                   │                                                    └Stack(alignment: AlignmentDirectional.topStart, fit: passthrough, overflow: clip, dependencies: [Directionality], renderObject: RenderStack#c5191)
flutter:                               │                   │                                                     ├IgnorePointer(ignoring: false, renderObject: RenderIgnorePointer#b3835)
flutter:                               │                   │                                                     │└RepaintBoundary-[GlobalKey#15dcb](renderObject: RenderRepaintBoundary#a5270)
flutter:                               │                   │                                                     │ └Builder
flutter:                               │                   │                                                     │  └Semantics(container: false, properties: SemanticsProperties, label: null, value: null, hint: null, hintOverrides: null, renderObject: RenderSemanticsAnnotations#d71a5)
flutter:                               │                   │                                                     │   └ExampleWidget(state: _ExampleWidgetState#28656)
flutter:                               │                   │                                                     │    └Column(direction: vertical, mainAxisAlignment: center, crossAxisAlignment: center, renderObject: RenderFlex#40f0f OVERFLOWING)
flutter:                               │                   │                                                     │     ├TextField(controller: TextEditingController#32cd0(TextEditingValue(text: ┤├, selection: TextSelection(baseOffset: -1, extentOffset: -1, affinity: TextAffinity.downstream, isDirectional: false), composing: TextRange(start: -1, end: -1))), decoration: InputDecoration(hintText: "Type something"), state: _TextFieldState#074f8)
flutter:                               │                   │                                                     │     │└ErrorWidget-[#87af5](
flutter:                               │                   │                                                     │     │  FlutterError:
flutter:                               │                   │                                                     │     │    No Material widget found.
flutter:                               │                   │                                                     │     │    TextField widgets require a Material widget ancestor.
flutter:                               │                   │                                                     │     │    In material design, most widgets are conceptually "printed" on a
flutter:                               │                   │                                                     │     │    sheet of material. In Flutter's material library, that material
flutter:                               │                   │                                                     │     │    is represented by the Material widget. It is the Material widget
flutter:                               │                   │                                                     │     │    that renders ink splashes, for instance. Because of this, many
flutter:                               │                   │                                                     │     │    material library widgets require that there be a Material widget
flutter:                               │                   │                                                     │     │    in the tree above them.
flutter:                               │                   │                                                     │     │    To introduce a Material widget, you can either directly include
flutter:                               │                   │                                                     │     │    one, or use a widget that contains Material itself, such as a
flutter:                               │                   │                                                     │     │    Card, Dialog, Drawer, or Scaffold.
flutter:                               │                   │                                                     │     │    The specific widget that could not find a Material ancestor was:
flutter:                               │                   │                                                     │     │      TextField
flutter:                               │                   │                                                     │     │    The ancestors of this widget were:
flutter:                               │                   │                                                     │     │      Column
flutter:                               │                   │                                                     │     │      ExampleWidget
flutter:                               │                   │                                                     │     │      Semantics
flutter:                               │                   │                                                     │     │      Builder
flutter:                               │                   │                                                     │     │      RepaintBoundary-[GlobalKey#15dcb]
flutter:                               │                   │                                                     │     │      IgnorePointer
flutter:                               │                   │                                                     │     │      Stack
flutter:                               │                   │                                                     │     │      _CupertinoBackGestureDetector<dynamic>
flutter:                               │                   │                                                     │     │      DecoratedBox
flutter:                               │                   │                                                     │     │      DecoratedBoxTransition
flutter:                               │                   │                                                     │     │      FractionalTranslation
flutter:                               │                   │                                                     │     │      SlideTransition
flutter:                               │                   │                                                     │     │      FractionalTranslation
flutter:                               │                   │                                                     │     │      SlideTransition
flutter:                               │                   │                                                     │     │      CupertinoPageTransition
flutter:                               │                   │                                                     │     │      AnimatedBuilder
flutter:                               │                   │                                                     │     │      RepaintBoundary
flutter:                               │                   │                                                     │     │      _FocusScopeMarker
flutter:                               │                   │                                                     │     │      Semantics
flutter:                               │                   │                                                     │     │      FocusScope
flutter:                               │                   │                                                     │     │      PageStorage
flutter:                               │                   │                                                     │     │      Offstage
flutter:                               │                   │                                                     │     │      _ModalScopeStatus
flutter:                               │                   │                                                     │     │      _ModalScope<dynamic>-[LabeledGlobalKey<_ModalScopeState<dynamic>>#ba876]
flutter:                               │                   │                                                     │     │      _OverlayEntry-[LabeledGlobalKey<_OverlayEntryState>#0d2eb]
flutter:                               │                   │                                                     │     │      Stack
flutter:                               │                   │                                                     │     │      _Theatre
flutter:                               │                   │                                                     │     │      Overlay-[LabeledGlobalKey<OverlayState>#fc378]
flutter:                               │                   │                                                     │     │      _FocusScopeMarker
flutter:                               │                   │                                                     │     │      Semantics
flutter:                               │                   │                                                     │     │      FocusScope
flutter:                               │                   │                                                     │     │      AbsorbPointer
flutter:                               │                   │                                                     │     │      Listener
flutter:                               │                   │                                                     │     │      Navigator-[GlobalObjectKey<NavigatorState> _WidgetsAppState#b29e3]
flutter:                               │                   │                                                     │     │      IconTheme
flutter:                               │                   │                                                     │     │      IconTheme
flutter:                               │                   │                                                     │     │      _InheritedCupertinoTheme
flutter:                               │                   │                                                     │     │      CupertinoTheme
flutter:                               │                   │                                                     │     │      _InheritedTheme
flutter:                               │                   │                                                     │     │      Theme
flutter:                               │                   │                                                     │     │      AnimatedTheme
flutter:                               │                   │                                                     │     │      Builder
flutter:                               │                   │                                                     │     │      DefaultTextStyle
flutter:                               │                   │                                                     │     │      CustomPaint
flutter:                               │                   │                                                     │     │      Banner
flutter:                               │                   │                                                     │     │      CheckedModeBanner
flutter:                               │                   │                                                     │     │      Title
flutter:                               │                   │                                                     │     │      Directionality
flutter:                               │                   │                                                     │     │      _LocalizationsScope-[GlobalKey#cf8d6]
flutter:                               │                   │                                                     │     │      Semantics
flutter:                               │                   │                                                     │     │      Localizations
flutter:                               │                   │                                                     │     │      MediaQuery
flutter:                               │                   │                                                     │     │      WidgetsApp-[GlobalObjectKey _MaterialAppState#3d235]
flutter:                               │                   │                                                     │     │      ScrollConfiguration
flutter:                               │                   │                                                     │     │      MaterialApp
flutter:                               │                   │                                                     │     │      MissingMaterialError
flutter:                               │                   │                                                     │     │      Semantics
flutter:                               │                   │                                                     │     │      Builder
flutter:                               │                   │                                                     │     │      RepaintBoundary-[GlobalKey#40410]
flutter:                               │                   │                                                     │     │      IgnorePointer
flutter:                               │                   │                                                     │     │      Stack
flutter:                               │                   │                                                     │     │      _CupertinoBackGestureDetector<dynamic>
flutter:                               │                   │                                                     │     │      DecoratedBox
flutter:                               │                   │                                                     │     │      DecoratedBoxTransition
flutter:                               │                   │                                                     │     │      FractionalTranslation
flutter:                               │                   │                                                     │     │      SlideTransition
flutter:                               │                   │                                                     │     │      FractionalTranslation
flutter:                               │                   │                                                     │     │      SlideTransition
flutter:                               │                   │                                                     │     │      CupertinoPageTransition
flutter:                               │                   │                                                     │     │      AnimatedBuilder
flutter:                               │                   │                                                     │     │      RepaintBoundary
flutter:                               │                   │                                                     │     │      _FocusScopeMarker
flutter:                               │                   │                                                     │     │      Semantics
flutter:                               │                   │                                                     │     │      FocusScope
flutter:                               │                   │                                                     │     │      PageStorage
flutter:                               │                   │                                                     │     │      Offstage
flutter:                               │                   │                                                     │     │      _ModalScopeStatus
flutter:                               │                   │                                                     │     │      _ModalScope<dynamic>-[LabeledGlobalKey<_ModalScopeState<dynamic>>#02ed6]
flutter:                               │                   │                                                     │     │      _OverlayEntry-[LabeledGlobalKey<_OverlayEntryState>#94a6c]
flutter:                               │                   │                                                     │     │      Stack
flutter:                               │                   │                                                     │     │      _Theatre
flutter:                               │                   │                                                     │     │      Overlay-[LabeledGlobalKey<OverlayState>#49f90]
flutter:                               │                   │                                                     │     │      _FocusScopeMarker
flutter:                               │                   │                                                     │     │      Semantics
flutter:                               │                   │                                                     │     │      FocusScope
flutter:                               │                   │                                                     │     │      AbsorbPointer
flutter:                               │                   │                                                     │     │      Listener
flutter:                               │                   │                                                     │     │      Navigator-[GlobalObjectKey<NavigatorState> _WidgetsAppState#8a10e]
flutter:                               │                   │                                                     │     │      IconTheme
flutter:                               │                   │                                                     │     │      IconTheme
flutter:                               │                   │                                                     │     │      _InheritedCupertinoTheme
flutter:                               │                   │                                                     │     │      CupertinoTheme
flutter:                               │                   │                                                     │     │      _InheritedTheme
flutter:                               │                   │                                                     │     │      Theme
flutter:                               │                   │                                                     │     │      AnimatedTheme
flutter:                               │                   │                                                     │     │      Builder
flutter:                               │                   │                                                     │     │      DefaultTextStyle
flutter:                               │                   │                                                     │     │      CustomPaint
flutter:                               │                   │                                                     │     │      Banner
flutter:                               │                   │                                                     │     │      CheckedModeBanner
flutter:                               │                   │                                                     │     │      Title
flutter:                               │                   │                                                     │     │      Directionality
flutter:                               │                   │                                                     │     │      _LocalizationsScope-[GlobalKey#3153c]
flutter:                               │                   │                                                     │     │      Semantics
flutter:                               │                   │                                                     │     │      Localizations
flutter:                               │                   │                                                     │     │      MediaQuery
flutter:                               │                   │                                                     │     │      WidgetsApp-[GlobalObjectKey _MaterialAppState#fe958]
flutter:                               │                   │                                                     │     │      ScrollConfiguration
flutter:                               │                   │                                                     │     │      MaterialApp
flutter:                               │                   │                                                     │     │      MyApp
flutter:                               │                   │                                                     │     │      [root]
flutter:                               │                   │                                                     │     │  , renderObject: RenderErrorBox#8a966)
flutter:                               │                   │                                                     │     └RaisedButton(dependencies: [_LocalizationsScope-[GlobalKey#cf8d6], _InheritedTheme])
flutter:                               │                   │                                                     │      └RawMaterialButton(state: _RawMaterialButtonState#2f6c3)
flutter:                               │                   │                                                     │       └Semantics(container: true, properties: SemanticsProperties, label: null, value: null, hint: null, hintOverrides: null, renderObject: RenderSemanticsAnnotations#6ebd1 relayoutBoundary=up1)
flutter:                               │                   │                                                     │        └_InputPadding(renderObject: _RenderInputPadding#8b6ee relayoutBoundary=up2)
flutter:                               │                   │                                                     │         └ConstrainedBox(BoxConstraints(88.0<=w<=Infinity, 36.0<=h<=Infinity), renderObject: RenderConstrainedBox#db908 relayoutBoundary=up3)
flutter:                               │                   │                                                     │          └Material(type: button, elevation: 2.0, color: Color(0xffe0e0e0), textStyle.debugLabel: ((englishLike button 2014).merge(blackCupertino button)).copyWith, textStyle.inherit: false, textStyle.color: Color(0xdd000000), textStyle.family: .SF UI Text, textStyle.size: 14.0, textStyle.weight: 500, textStyle.baseline: alphabetic, textStyle.decoration: TextDecoration.none, shape: RoundedRectangleBorder(BorderSide(Color(0xff000000), 0.0, BorderStyle.none), BorderRadius.circular(2.0)), state: _MaterialState#d4ce2)
```

Here's is the before output. Commas separating the blocks in the error message are a bit odd.
```
  │                                             └SlideTransition(animation: kAlwaysDismissedAnimation➩ProxyAnimation➩ProxyAnimation➩Cubic(0.35, 0.91, 0.33, 0.97)ₒₙ/Cubic(0.67, 0.03, 0.65, 0.09)➩Tween<Offset>(Offset(0.0, 0.0) → Offset(-0.3, 0.0))➩Offset(0.0, 0.0), state: _AnimatedState#d3855)
flutter:                               │                   │                                              └FractionalTranslation(renderObject: RenderFractionalTranslation#c6464)
flutter:                               │                   │                                               └SlideTransition(animation: AnimationController#b6924(⏭ 1.000; paused; for MaterialPageRoute<dynamic>(/))➩ProxyAnimation➩Cubic(0.35, 0.91, 0.33, 0.97)ₒₙ/Cubic(0.67, 0.03, 0.65, 0.09)➩Tween<Offset>(Offset(1.0, 0.0) → Offset(0.0, 0.0))➩Offset(0.0, 0.0), state: _AnimatedState#7398b)
flutter:                               │                   │                                                └FractionalTranslation(renderObject: RenderFractionalTranslation#ff656)
flutter:                               │                   │                                                 └DecoratedBoxTransition(animation: AnimationController#b6924(⏭ 1.000; paused; for MaterialPageRoute<dynamic>(/))➩ProxyAnimation➩Cubic(0.35, 0.91, 0.33, 0.97)➩DecorationTween(_CupertinoEdgeShadowDecoration(edgeGradient: null) → _CupertinoEdgeShadowDecoration(edgeGradient: LinearGradient(AlignmentDirectional(0.9, 0.0), AlignmentDirectional.centerEnd, [Color(0x00000000), Color(0x04000000), Color(0x12000000), Color(0x38000000)], [0.0, 0.3, 0.6, 1.0], TileMode.clamp)))➩_CupertinoEdgeShadowDecoration(edgeGradient: LinearGradient(AlignmentDirectional(0.9, 0.0), AlignmentDirectional.centerEnd, [Color(0x00000000), Color(0x04000000), Color(0x12000000), Color(0x38000000)], [0.0, 0.3, 0.6, 1.0], TileMode.clamp)), state: _AnimatedState#616ed)
flutter:                               │                   │                                                  └DecoratedBox(bg: _CupertinoEdgeShadowDecoration(edgeGradient: LinearGradient(AlignmentDirectional(0.9, 0.0), AlignmentDirectional.centerEnd, [Color(0x00000000), Color(0x04000000), Color(0x12000000), Color(0x38000000)], [0.0, 0.3, 0.6, 1.0], TileMode.clamp)), dependencies: [MediaQuery, _LocalizationsScope-[GlobalKey#cf8d6], Directionality], renderObject: RenderDecoratedBox#fe1a8)
flutter:                               │                   │                                                   └_CupertinoBackGestureDetector<dynamic>(dependencies: [MediaQuery, Directionality], state: _CupertinoBackGestureDetectorState<dynamic>#44cf9)
flutter:                               │                   │                                                    └Stack(alignment: AlignmentDirectional.topStart, fit: passthrough, overflow: clip, dependencies: [Directionality], renderObject: RenderStack#c5191)
flutter:                               │                   │                                                     ├IgnorePointer(ignoring: false, renderObject: RenderIgnorePointer#b3835)
flutter:                               │                   │                                                     │└RepaintBoundary-[GlobalKey#15dcb](renderObject: RenderRepaintBoundary#a5270)
flutter:                               │                   │                                                     │ └Builder
flutter:                               │                   │                                                     │  └Semantics(container: false, properties: SemanticsProperties, label: null, value: null, hint: null, hintOverrides: null, renderObject: RenderSemanticsAnnotations#d71a5)
flutter:                               │                   │                                                     │   └ExampleWidget(state: _ExampleWidgetState#28656)
flutter:                               │                   │                                                     │    └Column(direction: vertical, mainAxisAlignment: center, crossAxisAlignment: center, renderObject: RenderFlex#40f0f OVERFLOWING)
flutter:                               │                   │                                                     │     ├TextField(controller: TextEditingController#32cd0(TextEditingValue(text: ┤├, selection: TextSelection(baseOffset: -1, extentOffset: -1, affinity: TextAffinity.downstream, isDirectional: false), composing: TextRange(start: -1, end: -1))), decoration: InputDecoration(hintText: "Type something"), state: _TextFieldState#074f8)
flutter:                               │                   │                                                     │     │└ErrorWidget-[#04441](
flutter:                               │                   │                                                     │     │  No Material widget found.
flutter:                               │                   │                                                     │     │  ,
flutter:                               │                   │                                                     │     │  TextField widgets require a Material widget ancestor.
flutter:                               │                   │                                                     │     │  In material design, most widgets are conceptually "printed" on a
flutter:                               │                   │                                                     │     │  sheet of material. In Flutter's material library, that material
flutter:                               │                   │                                                     │     │  is represented by the Material widget. It is the Material widget
flutter:                               │                   │                                                     │     │  that renders ink splashes, for instance. Because of this, many
flutter:                               │                   │                                                     │     │  material library widgets require that there be a Material widget
flutter:                               │                   │                                                     │     │  in the tree above them.
flutter:                               │                   │                                                     │     │  ,
flutter:                               │                   │                                                     │     │  To introduce a Material widget, you can either directly include
flutter:                               │                   │                                                     │     │  one, or use a widget that contains Material itself, such as a
flutter:                               │                   │                                                     │     │  Card, Dialog, Drawer, or Scaffold.
flutter:                               │                   │                                                     │     │  ,
flutter:                               │                   │                                                     │     │  The specific widget that could not find a Material ancestor was:
flutter:                               │                   │                                                     │     │    TextField
flutter:                               │                   │                                                     │     │  ,
flutter:                               │                   │                                                     │     │  The ancestors of this widget were:
flutter:                               │                   │                                                     │     │    Column
flutter:                               │                   │                                                     │     │    ExampleWidget
flutter:                               │                   │                                                     │     │    Semantics
flutter:                               │                   │                                                     │     │    Builder
flutter:                               │                   │                                                     │     │    RepaintBoundary-[GlobalKey#15dcb]
flutter:                               │                   │                                                     │     │    IgnorePointer
flutter:                               │                   │                                                     │     │    Stack
flutter:                               │                   │                                                     │     │    _CupertinoBackGestureDetector<dynamic>
flutter:                               │                   │                                                     │     │    DecoratedBox
flutter:                               │                   │                                                     │     │    DecoratedBoxTransition
flutter:                               │                   │                                                     │     │    FractionalTranslation
flutter:                               │                   │                                                     │     │    SlideTransition
flutter:                               │                   │                                                     │     │    FractionalTranslation
flutter:                               │                   │                                                     │     │    SlideTransition
flutter:                               │                   │                                                     │     │    CupertinoPageTransition
flutter:                               │                   │                                                     │     │    AnimatedBuilder
flutter:                               │                   │                                                     │     │    RepaintBoundary
flutter:                               │                   │                                                     │     │    _FocusScopeMarker
flutter:                               │                   │                                                     │     │    Semantics
flutter:                               │                   │                                                     │     │    FocusScope
flutter:                               │                   │                                                     │     │    PageStorage
flutter:                               │                   │                                                     │     │    Offstage
flutter:                               │                   │                                                     │     │    _ModalScopeStatus
flutter:                               │                   │                                                     │     │    _ModalScope<dynamic>-[LabeledGlobalKey<_ModalScopeState<dynamic>>#ba876]
flutter:                               │                   │                                                     │     │    _OverlayEntry-[LabeledGlobalKey<_OverlayEntryState>#0d2eb]
flutter:                               │                   │                                                     │     │    Stack
flutter:                               │                   │                                                     │     │    _Theatre
flutter:                               │                   │                                                     │     │    Overlay-[LabeledGlobalKey<OverlayState>#fc378]
flutter:                               │                   │                                                     │     │    _FocusScopeMarker
flutter:                               │                   │                                                     │     │    Semantics
flutter:                               │                   │                                                     │     │    FocusScope
flutter:                               │                   │                                                     │     │    AbsorbPointer
flutter:                               │                   │                                                     │     │    Listener
flutter:                               │                   │                                                     │     │    Navigator-[GlobalObjectKey<NavigatorState> _WidgetsAppState#b29e3]
flutter:                               │                   │                                                     │     │    IconTheme
flutter:                               │                   │                                                     │     │    IconTheme
flutter:                               │                   │                                                     │     │    _InheritedCupertinoTheme
flutter:                               │                   │                                                     │     │    CupertinoTheme
flutter:                               │                   │                                                     │     │    _InheritedTheme
flutter:                               │                   │                                                     │     │    Theme
flutter:                               │                   │                                                     │     │    AnimatedTheme
flutter:                               │                   │                                                     │     │    Builder
flutter:                               │                   │                                                     │     │    DefaultTextStyle
flutter:                               │                   │                                                     │     │    CustomPaint
flutter:                               │                   │                                                     │     │    Banner
flutter:                               │                   │                                                     │     │    CheckedModeBanner
flutter:                               │                   │                                                     │     │    Title
flutter:                               │                   │                                                     │     │    Directionality
flutter:                               │                   │                                                     │     │    _LocalizationsScope-[GlobalKey#cf8d6]
flutter:                               │                   │                                                     │     │    Semantics
flutter:                               │                   │                                                     │     │    Localizations
flutter:                               │                   │                                                     │     │    MediaQuery
flutter:                               │                   │                                                     │     │    WidgetsApp-[GlobalObjectKey _MaterialAppState#3d235]
flutter:                               │                   │                                                     │     │    ScrollConfiguration
flutter:                               │                   │                                                     │     │    MaterialApp
flutter:                               │                   │                                                     │     │    MissingMaterialError
flutter:                               │                   │                                                     │     │    Semantics
flutter:                               │                   │                                                     │     │    Builder
flutter:                               │                   │                                                     │     │    RepaintBoundary-[GlobalKey#40410]
flutter:                               │                   │                                                     │     │    IgnorePointer
flutter:                               │                   │                                                     │     │    Stack
flutter:                               │                   │                                                     │     │    _CupertinoBackGestureDetector<dynamic>
flutter:                               │                   │                                                     │     │    DecoratedBox
flutter:                               │                   │                                                     │     │    DecoratedBoxTransition
flutter:                               │                   │                                                     │     │    FractionalTranslation
flutter:                               │                   │                                                     │     │    SlideTransition
flutter:                               │                   │                                                     │     │    FractionalTranslation
flutter:                               │                   │                                                     │     │    SlideTransition
flutter:                               │                   │                                                     │     │    CupertinoPageTransition
flutter:                               │                   │                                                     │     │    AnimatedBuilder
flutter:                               │                   │                                                     │     │    RepaintBoundary
flutter:                               │                   │                                                     │     │    _FocusScopeMarker
flutter:                               │                   │                                                     │     │    Semantics
flutter:                               │                   │                                                     │     │    FocusScope
flutter:                               │                   │                                                     │     │    PageStorage
flutter:                               │                   │                                                     │     │    Offstage
flutter:                               │                   │                                                     │     │    _ModalScopeStatus
flutter:                               │                   │                                                     │     │    _ModalScope<dynamic>-[LabeledGlobalKey<_ModalScopeState<dynamic>>#02ed6]
flutter:                               │                   │                                                     │     │    _OverlayEntry-[LabeledGlobalKey<_OverlayEntryState>#94a6c]
flutter:                               │                   │                                                     │     │    Stack
flutter:                               │                   │                                                     │     │    _Theatre
flutter:                               │                   │                                                     │     │    Overlay-[LabeledGlobalKey<OverlayState>#49f90]
flutter:                               │                   │                                                     │     │    _FocusScopeMarker
flutter:                               │                   │                                                     │     │    Semantics
flutter:                               │                   │                                                     │     │    FocusScope
flutter:                               │                   │                                                     │     │    AbsorbPointer
flutter:                               │                   │                                                     │     │    Listener
flutter:                               │                   │                                                     │     │    Navigator-[GlobalObjectKey<NavigatorState> _WidgetsAppState#8a10e]
flutter:                               │                   │                                                     │     │    IconTheme
flutter:                               │                   │                                                     │     │    IconTheme
flutter:                               │                   │                                                     │     │    _InheritedCupertinoTheme
flutter:                               │                   │                                                     │     │    CupertinoTheme
flutter:                               │                   │                                                     │     │    _InheritedTheme
flutter:                               │                   │                                                     │     │    Theme
flutter:                               │                   │                                                     │     │    AnimatedTheme
flutter:                               │                   │                                                     │     │    Builder
flutter:                               │                   │                                                     │     │    DefaultTextStyle
flutter:                               │                   │                                                     │     │    CustomPaint
flutter:                               │                   │                                                     │     │    Banner
flutter:                               │                   │                                                     │     │    CheckedModeBanner
flutter:                               │                   │                                                     │     │    Title
flutter:                               │                   │                                                     │     │    Directionality
flutter:                               │                   │                                                     │     │    _LocalizationsScope-[GlobalKey#3153c]
flutter:                               │                   │                                                     │     │    Semantics
flutter:                               │                   │                                                     │     │    Localizations
flutter:                               │                   │                                                     │     │    MediaQuery
flutter:                               │                   │                                                     │     │    WidgetsApp-[GlobalObjectKey _MaterialAppState#fe958]
flutter:                               │                   │                                                     │     │    ScrollConfiguration
flutter:                               │                   │                                                     │     │    MaterialApp
flutter:                               │                   │                                                     │     │    MyApp
flutter:                               │                   │                                                     │     │    [root]
flutter:                               │                   │                                                     │     │  , renderObject: RenderErrorBox#518b1)
flutter:                               │                   │                                                     │     └RaisedButton(dependencies: [_LocalizationsScope-[GlobalKey#cf8d6], _InheritedTheme])
```